oci8/oci8v3.php

```diff
@@ -0,0 +1,2710 @@
+     * Empty objects have zero length.
+     */
+    public function size() {}
+
+    /**
+     * (PHP 5, PECL OCI8 &gt;= 1.1.0)<br/>
+     * Alias of {@see OCILob::export}
+     * @link https://php.net/manual/en/oci-lob.writetofile.php
+     * @param $filename
+     * @param $start [optional]
+     * @param $length [optional]
+     * @return bool TRUE on success or FALSE on failure.
+     */
+    public function writetofile($filename, $start, $length) {}
+
+    /**
+     * (PHP 5, PECL OCI8 &gt;= 1.1.0)<br/>
+     * Exports LOB's contents to a file
+     * @link https://php.net/manual/en/oci-lob.export.php
+     * @param string $filename <p>
+     * Path to the file.
+     * </p>
+     * @param int $start [optional] <p>
+     * Indicates from where to start exporting.
+     * </p>
+     * @param int $length [optional] <p>
+     * Indicates the length of data to be exported.
+     * </p>
+     * @return bool <b>TRUE</b> on success or <b>FALSE</b> on failure.
+     */
+    public function export($filename, $start = null, $length = null) {}
+
+    /**
+     * (PHP 5, PECL OCI8 &gt;= 1.1.0)<br/>
+     * Imports file data to the LOB
+     * @link https://php.net/manual/en/oci-lob.import.php
+     * @param string $filename <p>
+     * Path to the file.
+     * </p>
+     * @return bool <b>TRUE</b> on success or <b>FALSE</b> on failure.
+     */
+    public function import($filename) {}
+
+    /**
+     * (PHP 5, PECL OCI8 &gt;= 1.1.0)<br/>
+     * Writes a temporary large object
+     * @link https://php.net/manual/en/oci-lob.writetemporary.php
+     * @param string $data <p>
+     * The data to write.
+     * </p>
+     * @param int $lob_type [optional] <p>
+     * Can be one of the following:
+     * <b>OCI_TEMP_BLOB</b> is used to create temporary BLOBs
+     * <b>OCI_TEMP_CLOB</b> is used to create
+     * temporary CLOBs
+     * </p>
+     * @return bool <b>TRUE</b> on success or <b>FALSE</b> on failure.
+     */
+    public function writeTemporary($data, $lob_type = OCI_TEMP_CLOB) {}
+
+    /**
+     * (PHP 5, PECL OCI8 &gt;= 1.1.0)<br/>
+     * Closes LOB descriptor
+     * @link https://php.net/manual/en/oci-lob.close.php
+     * @return bool <b>TRUE</b> on success or <b>FALSE</b> on failure.
+     */
+    public function close() {}
+
+    /**
+     * (PHP 5, PECL OCI8 &gt;= 1.1.0)<br/>
+     * Saves data to the large object
+     * @link https://php.net/manual/en/oci-lob.save.php
+     * @param string $data <p>
+     * The data to be saved.
+     * </p>
+     * @param int $offset [optional] <p>
+     * Can be used to indicate offset from the beginning of the large object.
+     * </p>
+     * @return bool <b>TRUE</b> on success or <b>FALSE</b> on failure.
+     */
+    public function save($data, $offset = null) {}
+
+    /**
+     * (PHP 5, PECL OCI8 &gt;= 1.1.0)<br/>
+     * Alias of {@see OCILob::import}
+     * @link https://php.net/manual/en/oci-lob.savefile.php
+     * @param $filename
```

Comment:
```suggestion
     * @param string $filename
```
Could you check for any other `@param $` block to add the types please ?